### ISSUE TITLE
asImageWidthSrcSet does not accept ImageField

### ISSUE THREAD

**Issue by benwest** [prismicio/prismic-types#30]
`asImageWidthSrcSet` expects an ImageFieldImage, but in the [docs](https://prismic.io/docs/technical-reference/prismicio-helpers#asimagewidthsrcset) it appears to be accepting an ImageField.

### Reproduction
```ts
import { asImageWidthSrcSet } from "@prismicio/helpers"
import * as prismicT from "@prismicio/types"

const image = {} as prismicT.ImageField
asImageWidthSrcSet(image) // Argument of type 'ImageField<never, FieldState>' is not assignable to parameter of type 'EmptyImageFieldImage | FilledImageFieldImage'.
```

**Comment by lihbr** [comment]
Hey there, thank you so much for the report again!

We're facing some TypeScript limitations with this type indeed... We'll attempt at fixing it without introducing further regression again!

**Comment by lihbr** [comment]
Fixed in `0.1.27` 🎉

Updating `@prismicio/types` should fix the issue, let us know if you still encounter issues 🙏
